### PR TITLE
issue#5 -> text edited, graphics changed above the fold, new card section with lightbox added below the fold, gift order front-end changed temporarily to test order

### PR DESCRIPTION
**Pull request by kolouchtomas**
Completed almost everything from issue#5, potentially will continue on wednesday

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -990,16 +1033,73 @@ section {
...
-    margin: auto;
-    display: block;
+.foto_reference img:not(.rozvijime__cards__picture) {
+	width: 90%;
+	margin: auto;
+	display: block;
+}
+
+.foto_reference__cards a:not(.button-fotogalerie):not(.button-video) {
+	width: 33.33%;
```
Comment (iamladi) [inline css/index.css:1087]: We need to make sure that your spaces match projects space. In our case your editor is set to 4 spaces per tab, but the project is set to 2. Without this, it's hard to read the code without being distracted by the different spaces.

File: js/index.js [hunk 2]
```diff
@@ -139,3 +139,69 @@ if (typeof zenscroll != 'undefined') {
   var edgeOffset = 56; // px
   zenscroll.setup(defaultDuration, edgeOffset);
 }
+
+/* added by Tom Kolouch */
```
Comment (iamladi) [inline js/index.js:143]: This attribution isn't necessary as Git blame can tell us who did this. ;)

**Review by iamladi — APPROVED** [review]

File: css/index.css (hunk 1 above)
Comment (kolouchtomas) [reply css/index.css:1087]: It is not complicated in my editor. I will solve this better next time

File: js/index.js (hunk 2 above)
Comment (kolouchtomas) [reply js/index.js:143]: OK, just old bad habits :-) will not pass any attribution like this into code in future

**Pull request merged**